python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -604,18 +616,21 @@ def __init__(self):
         """
         super(GangaObject, self).__init__(None)
 
-        self._data_dict = {}
         self._index_cache_dict = {}
         self._registry = None
 
         #Node.__init__(self, None)
 
         if self._schema is not None and hasattr(self._schema, 'allItems'):
+            self._data_dict = dict.fromkeys(self._schema.datadict)
```

Comment:
It's much more efficient than adding objects to a dictionary and when you're doing this a lot I thought it's worth the few cycles that we save per object.